src/main/java/com/dev/cinema/Main.java

```diff
@@ -49,8 +58,12 @@ public static void main(String[] args) {
         session.setMovie(fastAndFurious);
         session.setShowTime(LocalDateTime.now().minusDays(2));
         sessionService.add(session);
-
         sessionService.findAvailableSessions(1L, LocalDate.now())
                 .forEach(System.out::println);
+
+        authenticationService.register("bob@gmail.com", "1111");
+        authenticationService.register("alisa@gmail.com", "1111");
+        User bob = authenticationService.login("bob@gmail.com", "1111");
+        User alisa = authenticationService.login("alisa@gmail.com", "1111");
```

Comment:
I think we can check here is a user added with repeatable email as well

Reply:
I've checked. But then removed because of exception. Not sure, do we mast to keep this check.